frontend/src/app/organization/organization-offers-list/organization-offers-list.component.html

```diff
@@ -0,0 +1,49 @@
+<div class="container" *ngIf="offers; else noOffers;">
```

Comment:
*ngIf= "offers.length; ; else noOffers;"

Offers is an array, it exists even when it is empty (when there is no offers). Check for offers.length is true when there are offers inside the array (lenght is larger then 0).